test/io/GeantImporter.test.cc

```diff
@@ -71,3 +92,79 @@ TEST_F(GeantImporterTest, import_particles)
     EXPECT_VEC_EQ(expected_loaded_names, loaded_names);
     EXPECT_VEC_EQ(expected_loaded_pdgs, loaded_pdgs);
 }
+
+//---------------------------------------------------------------------------//
+TEST_F(GeantImporterTest, import_tables)
+{
+    GeantImporter import(root_filename_.c_str());
+    auto          data = import();
+
+    EXPECT_GE(data.physics_tables->size(), 0);
+
+    bool lambda_kn_gamma_table = false;
+
+    for (auto table : *data.physics_tables)
+    {
+        EXPECT_GE(table.physics_vectors.size(), 0);
+
+        if (table.particle == PDGNumber{celeritas::pdg::gamma()}
+            && table.table_type == GeantTableType::Lambda
+            && table.process == GeantProcess::compt
+            && table.model == GeantModel::KleinNishina)
+        {
+            lambda_kn_gamma_table = true;
+            break;
+        }
+    }
+    EXPECT_TRUE(lambda_kn_gamma_table);
+}
+
+//---------------------------------------------------------------------------//
+TEST_F(GeantImporterTest, import_geometry)
+{
+    GeantImporter import(root_filename_.c_str());
+    auto          data = import();
+
+    auto map = data.geometry->volid_to_matid_map();
+    EXPECT_EQ(map.size(), 4257);
+
+    // Fetch a given GeantVolume provided a vol_id
+    GeantGeometryMap::vol_id volid  = 10;
+    GeantVolume              volume = data.geometry->get_volume(volid);
+    EXPECT_EQ(volume.name, "TrackerPatchPannel");
+
+    // Fetch respective mat_id and GeantMaterial from the given vol_id
+    auto          matid    = data.geometry->get_matid(volid);
+    GeantMaterial material = data.geometry->get_material(matid);
+
+    // Material
+    EXPECT_EQ(matid, 31);
+    EXPECT_EQ(material.name, "Air");
+    EXPECT_EQ(material.state, MaterialState::gas);
+    EXPECT_SOFT_EQ(material.temperature, 293.15);
+    EXPECT_SOFT_EQ(material.density, 0.00121399936124299);
+    EXPECT_SOFT_EQ(material.electron_density, 3.6523656201748414e+20);
+    EXPECT_SOFT_EQ(material.atomic_density, 5.0756589772243755e+19);
+    EXPECT_SOFT_EQ(material.radiation_length, 30152.065419629631);
+    EXPECT_SOFT_EQ(material.nuclear_int_length, 70408.106699294673);
+    EXPECT_EQ(material.elements.size(), 4);
+
+    // Elements within material
+    std::string elements_name[4] = {"N", "O", "Ar", "H"};
+    real_type   fraction[4]      = {0.7494, 0.2369, 0.0129, 0.0008};
+    int         atomic_number[4] = {7, 8, 18, 1};
+    real_type   atomic_mass[4]
+        = {14.00676896, 15.999390411, 39.94769335110001, 1.007940752665138};
+
+    for (size_t i = 0; i < 4; i++)
```

Comment:
`size_t` isn't (in the C++ standard) in the global namespace so it should be scoped as `std::size_t`; or you can just say `for (auto i : celeritas::range(4))`.

I would also change the loop criterion from `4` to `material.elements.size()` for additional clarity.

Reply:
Yes, this was an autocomplete mistake that I missed, since my original thought was to write size_type. I replaced that by a `celeritas::range()` statement.